### DISCUSSION TITLE
Add "View" view for each element to display input data after submission

### DISCUSSION THREAD

**Discussion by adamberecz** [vueform/vueform#117] [Ideas]
Add "View" view for each element to display input data after submission

**Comment by nclemeur** [comment]
Could we switch easily from a 'view' (readonly) mode to a normal edit mode? That would indeed be a great feature.

**Reply by adamberecz** [reply]
Yes, that would be the idea.

**Reply by jensjns** [reply]
This is something I'm looking forward to aswell! 👀
